azurerm/azurerm_windows_virtual_machine/simple/main.tf

```diff
@@ -0,0 +1,88 @@
+variable "azure_subscription_id" {
+  type = string
+}
+
+# Documentation: https://www.terraform.io/docs/language/providers/requirements.html
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs
+provider "azurerm" {
+  features {}
+
+  subscription_id = var.azure_subscription_id
+}
+
+# Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group
+resource "azurerm_resource_group" "changeme_simple_windows_virtual_machine_resource_group" {
+  name     = "changeme-simple-windows-virtual-machine-resource-group-name"
+  location = "West Europe"
+}
+
+# Virtual Network within the Resource Group
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/virtual_network
+resource "azurerm_virtual_network" "changeme_simple_windows_virtual_machine_virtual_network" {
+  name                = "changeme-simple-windows-virtual-machine-virtual-network-name"
+  resource_group_name = azurerm_resource_group.changeme_simple_windows_virtual_machine_resource_group.name
+  location            = azurerm_resource_group.changeme_simple_windows_virtual_machine_resource_group.location
+  address_space       = ["10.0.0.0/16"]
+}
+
+# Subnet within the Virtual Network
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet
+resource "azurerm_subnet" "changeme_simple_windows_virtual_machine_subnet" {
+  name                 = "changeme-simple-windows-virtual-machine-subnet-name"
+  resource_group_name  = azurerm_resource_group.changeme_simple_windows_virtual_machine_resource_group.name
+  virtual_network_name = azurerm_virtual_network.changeme_simple_windows_virtual_machine_virtual_network.name
+  address_prefixes     = ["10.0.1.0/24"]
+}
+
+# NIC attached to the Subnet
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/network_interface
+resource "azurerm_network_interface" "changeme_simple_windows_virtual_machine_network_interface" {
+  name                = "changeme-simple-windows-virtual-machine-network-interface-name"
+  location            = azurerm_resource_group.changeme_simple_windows_virtual_machine_resource_group.location
+  resource_group_name = azurerm_resource_group.changeme_simple_windows_virtual_machine_resource_group.name
+
+  ip_configuration {
+    name                          = "internal"
+    subnet_id                     = azurerm_subnet.changeme_simple_windows_virtual_machine_subnet.id
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+# Windows Virtual Machine with the NIC attached
+# Documentation: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/windows_virtual_machine
+resource "azurerm_windows_virtual_machine" "changeme_simple_windows_virtual_machine" {
+  name                  = "changeme-simple-windows-virtual-machine-name"
```

Comment:
CI job `ProvidersAzure` failed to do `terraform apply` here:
```
azurerm_windows_virtual_machine.changeme_simple_windows_virtual_machine: Creating...
╷
│ Error: unable to assume default computer name "computer_name" can be at most 15 characters, got 44. Please adjust the "name", or specify an explicit "computer_name"
│ 
│   with azurerm_windows_virtual_machine.changeme_simple_windows_virtual_machine,
│   on main.tf line 68, in resource "azurerm_windows_virtual_machine" "changeme_simple_windows_virtual_machine":
│   68: resource "azurerm_windows_virtual_machine" "changeme_simple_windows_virtual_machine" ***
│ 
╵
/home/runner/work/terraform-examples/terraform-examples/azurerm/azurerm_windows_virtual_machine/simple
```

Please make sure that `./run.sh` works correctly for all new examples.

Reply:
Fixed: https://github.com/ContainerSolutions/terraform-examples/pull/62/commits/bdf06885c36c1b850c594dfb220649df66c2962f